RC SAML SSO updates (#1994)
RedisLabs/redislabs-docs: content/rc/security/single-sign-on/social-login.md

---
Title: Social login
linkTitle: Social login
description: Social login with Redis Cloud.
weight: 50
alwaysopen: false
categories: ["RC"]
---

> One single sign-on option that Redis Cloud offers is [social login](https://en.wikipedia.org/wiki/Social_login), which lets you use an existing social media account to create or sign in to your Redis Cloud account.

Reviewer: ```suggestion
Redis Cloud offers [social login](https://en.wikipedia.org/wiki/Social_login) as a single sign-on (SSO) option, which lets you use an existing social media account to create or sign in to your Redis Cloud account.
```
